vembrane/modules/filter.py

```diff
@@ -16,6 +16,14 @@
 from ..representations import Environment
 from .. import __version__
 
+from argparse import Action
+
+
+class StoreMapping(Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        d = dict(m.split("=") for m in values.split(","))
+        setattr(namespace, self.dest, d)
```

Comment:
```suggestion
        items = getattr(namespace, self.dest, None)
        if items is self.default or items is None:
            items = {}
        items.update(dict(m.split("=") for m in values.split(",")))
        setattr(namespace, self.dest, items)
```
In case you want to
- add a `default` value at some point (the logic above would discard the `default` content completely when the user provided anything),
- support multiple arguments, e.g., `vembrane --overwrite-number=X=. --overwrite-number=Y=. ...` -> `{"X": ".", "Y": "."}`.

Reply:
Also, if you want to go by "also I guess you hopefully won't have many things to overwrite...", you could also omit the `,`-splitting logic for now -- since it's a relaxation, it could be added later at any point in time, if it does seem convenient to include.

Reply:
Thanks for the update logic :)